examples/eyalroz_printf/config.cmake

```diff
@@ -0,0 +1,6 @@
+hunter_config(
+    eyalroz_printf 
+    VERSION 6.2.0-b1 
```

Comment:
Sind hardcode the version number in the config file. Use the HUNTER_eyalroz_printf_VERSION variable instead